MIGRATING.md

```diff
@@ -36,26 +48,88 @@ Follow these steps to remove the tracing library and its dependencies:
     gem uninstall signalfx
     ```
 
-2.  Remove `signalfx` from your Gemfile.
+1.  Remove `signalfx` from your Gemfile.
 
-3.  Remove any additional OpenTracing instrumentation packages you
-    installed.
+### Deploy the Splunk Ruby agent
+
+To install the Splunk Distribution of OpenTelemetry Ruby, see the [README.md](README.md).
+
+## Migrate to the Splunk Distribution of OpenTelemetry Ruby
+
+We're currently only testing installation using Bundler with RubyGems as a source.
+[Contact us](mailto:ssg-observability-instrumentals-ruby@splunk.com) if you require a different way of installation.
+
+If you installed
+[Smart Agent](https://github.com/signalfx/signalfx-agent)
+to serve as a gateway (traces/metrics data proxy) then,
+if possible, migrate to
+[OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/resources.html)
+first.
```

Comment:
Suggestion: If you installed [Smart Agent](https://github.com/signalfx/signalfx-agent)
to serve as a gateway (traces/metrics data proxy), migrate to
[OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/resources.html)
first, as soon as (if) possible.